### DISCUSSION TITLE
Easy entry of new network info in existing BirdNET-Pi

### DISCUSSION THREAD

**Discussion by tip184** [mcguirepr89/BirdNET-Pi#1133] [Ideas]
I want to give BirdNET-Pis as gifts to some friends whose network info is unknown to me.  I cannot see a way to give them workable devices without this info. 

Is there an easy way that a non-geek could plug in and boot the BirdNET-PI then enter his network SSID and password so that the BirdNET-Pi could use it in the future? I know how to use the web terminal to use raspi-config, but I think this requires the Pi to be logged on to a network.

Apologies if this has been addressed previously -- a (maybe cursory) search found nothng.

Thanks!

**Comment by pddpauw** [comment]
you could set the Pi up with a fixed IP, and then connect with lan cable and a laptop with a fixed ip in the same subnet....  Would still require them to log in and select network with raspi-config... (or you could send a command from the laptopt to that Pi with a script that asks for SSID and pwd from the user).

Alternatively, you could preconfigure that wifi (but you would need the pwd in that case).

**Reply by stevedee** [reply]
You could initially config the Pi with a simple access point name & password, using a temporary hotspot on your phone.
The recipient then creates the same temporary hotspot on their phone and uses this to access the BirdNet-Pi via a laptop.
They then run a command to update or add their real AP & password to wpa_supplicant.

If this already sounds too geekie, I won't flesh it out any further!

**Reply by tip184** [reply]
I appreciate both responses. I could do these things, but the recipients are unlikely to have the geek skills required. What I would love is a plug-n-play possibility. Maybe have the Pi create its own hotspot that the user could access via phone or laptop. The Pi then prompts for the user's netwrok name and password and saves them itself. I am NOT geeky enough to figure out how to make this happen, but it seems that it should be possible.

I think I will simply ask for network name and password. That way I'll find out who trusts me and who doesn't.

**Reply by stevedee** [reply]
If the Pi was set up to provide a hotspot, wouldn't that be enough?
The user could just connect via phone/laptop whenever they wanted to check BirdNet data.

**Reply by tip184** [reply]
Yes, but if I undersrtand the BirdNET program it wouldneed ot be connected to the internet to access the Cornell and Wiki sites that it uses, and to be part of Birdweather.
